cmd/celestia-appd/cmd/migrate_config.go

```diff
@@ -0,0 +1,214 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v6/app"
+	"github.com/celestiaorg/celestia-app/v6/pkg/appconsts"
+	"github.com/cometbft/cometbft/config"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	serverconfig "github.com/cosmos/cosmos-sdk/server/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// ConfigUpdater defines a function type for applying version-specific updates
+type ConfigUpdater func(*config.Config, *serverconfig.Config) (*config.Config, *serverconfig.Config)
+
+// updateRegistry maps version strings to their corresponding update functions
+var updateRegistry = map[string]ConfigUpdater{
+	"v6": applyV6Config,
+}
+
+// updateConfigCmd returns the update-config command that updates
+// configuration files based on the target version.
+func updateConfigCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "update-config",
+		Short:   "Update configuration values to be that of a specific app version",
+		Long:    "Update configuration files (config.toml and app.toml) to be compatible with a specific app version.",
+		Example: "celestia-appd update-config --home ~/.celestia-app\ncelestia-appd update-config --version v6 --home ~/.celestia-app",
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			homeDir, err := cmd.Flags().GetString(flags.FlagHome)
```

Comment:
I think there is an app.NodeHome that you can use instead. At least that's what I did with the min gas migration if you want to have a look at it. That way users don't have to provide it if it's in the default location